src/Microsoft.DotNet.ApiSymbolExtensions/Filtering/SymbolAccessibilityBasedFilter.cs

```diff
@@ -2,22 +2,25 @@
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
 using Microsoft.CodeAnalysis;
-using Microsoft.DotNet.ApiSymbolExtensions;
 
 namespace Microsoft.DotNet.ApiSymbolExtensions
 {
     public class SymbolAccessibilityBasedFilter : ISymbolFilter
     {
         private readonly bool _includeInternalSymbols;
         private readonly bool _includeEffectivelyPrivateSymbols;
+        private readonly bool _includeExplicitInterfaceImplementationSymbols;
 
-        public SymbolAccessibilityBasedFilter(bool includeInternalSymbols, bool includeEffectivelyPrivateSymbols = false)
+        public SymbolAccessibilityBasedFilter(bool includeInternalSymbols,
+            bool includeEffectivelyPrivateSymbols = false, bool includeExplicitInterfaceImplementationSymbols = false)
```

Comment:
nit: we usually declare either all or just one parameter per line

Reply:
Updated